android/src/main/kotlin/io/dolby/comms/sdk/flutter/NativeView.kt

```diff
@@ -71,12 +71,10 @@ class NativeView(context: Context, id: Int, messenger: BinaryMessenger): Platfor
     }
 
     fun setScaleType(scaleType: String?) {
```

Comment:
Does `scaletype: String?` need to be an optional?

Reply:
As far I can see all of creationParameters read from flutter are nullable.

Reply:
Ok. I will resolve the conversation.